pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -10909,6 +10909,115 @@
+        get_coding: "Direct your students to the activity by sharing the link on the board or through your classroom's learning management system. Your students may encounter challenges—that's okay! Learning to program is like learning a new language; you won't be fluent right away. If you don't know the answer, you can figure it out together."
+        celebrate: "After your event, print certificates of completion and congratulate your students. Share photos and videos of your Hour of Code event and show off your students' achievements using #HourOfCode and @codeorg."
+        beyond:
+          body_01: "It doesn't end here!"
+          body_02: "Computer science doesn't have to end with the Hour of Code! While 90% of parents want their child to learn computer science, most schools still don't teach it. Check out the diverse curriculum offerings from Hour of Code partners, and learn how to bring CS to your school today!"
+    events:
+      heading: "Events"
+      top:
+        heading: "How to organize an Hour of Code event"
+        desc_01: "Hour of Code is a one-hour introduction to computer science, using fun and interactive tutorials to show that anyone can learn the basics. Your efforts can make a huge impact on the way students view computer science and their own potential."
+        desc_02: "Schools and organizations who fully engage with Hour of Code can help raise awareness of the computer science movement and inspire more students to try computer science, particularly young women and students from historically marginalized racial and ethnic groups."
+      steps:
+        register: "Register your Hour of Code event to receive information about new activities and special promotions!"
+        choose_activity: "Explore the coding activity library and choose one based on your group's age, experience, interests and more."
+        prepare:
+          list_01: "Consider where and when you want to hold your event, as well as the tech capabilities of your space. Students can work on their own devices, in pairs on one device, or together as a classroom on a projector, and there are even unplugged tutorials that can be completed with no devices at all!"
+          list_02: "Send an email to an elected official or influential business person and invite them to speak."
+          list_03: "You may also invite a local politician or local media, such as a local news station, newspaper, or education/tech bloggers."
+        plan:
+          body_01: "Here's a sample agenda to get you started:"
+          list_01: "10:00-10:05am: Open with one of our [inspirational video](%{video_url})."
+          list_02: "10:05-10:15am: Principal gives an intro overviewing the importance of computer science."
+          list_03: "10:15-10:30am: Special guest to talk about their background and how technology and computer science plays an important role in their everyday lives"
```

Comment:
tiny nit: should this have a period at the end?

Reply:
Added!